fdbcli/linenoise/linenoise.c

```diff
@@ -456,6 +471,32 @@ static void abFree(struct abuf *ab) {
     free(ab->b);
 }
 
+/* Helper of refreshSingleLine() and refreshMultiLine() to show hints
+ * to the right of the prompt. */
+void refreshShowHints(struct abuf *ab, struct linenoiseState *l, int plen) {
+    char seq[64];
+    if (hintsCallback && plen+l->len < l->cols) {
```

Comment:
I'm not sure clang-format would allows `plen+l->len` -- did you run it?

Reply:
I am inclined to not clang-format third party code, because it will make future merges with upstream harder.